Add recommended citation format to README template

chuehlien opened the pull request:
[RDAODTD-2294](https://usdotjpoode.atlassian.net/browse/RDAODTD-2294)

Added recommended citations/acknowledgments to ITS CodeHub README template to make it easier to track the reuse of JPO-funded code as well as promote proper citation/attribution.

chuehlien committed:
Add recommended citation format to README template

eberlyn commented on README.md:
```diff
@@ -144,4 +144,13 @@ To track how this government-funded code is used, we request that if you decide
 
 Digital Object Identifier: (fill in with DOI)
 
+To cite this code in a publication or report, please cite our associated report/paper and/or our source code:
```
. Below is a sample citation for this code:

eberlyn commented on README.md:
```diff
@@ -144,4 +144,13 @@ To track how this government-funded code is used, we request that if you decide
+To cite this code in a publication or report, please cite our associated report/paper and/or our source code:
+(Fill out the `formatted fields` in the below citation with details of your source code)
+
+`author_surname_or_organization`, `initial`. (`year`). _`program_or_source_code_title`_ (`code_version`) [Source code]. Provided by ITS CodeHub through GitHub.com. Accessed YYYY-MM-DD from `doi_url`.
```
What initial is 'initial' supposed to be?

chuehlien replied:
Hm we could reword this. It's suppose to mean that when you list individual contributor names, it should follow the format of Last Name, First Name Initial. (e.g. Lien, C.).

eberlyn replied:
Yeah, it seems a bit confusing since we say they can use the organization name. At the least, I would name it 'first initial.'

eberlyn commented on README.md:
```diff
@@ -144,4 +144,13 @@ To track how this government-funded code is used, we request that if you decide
+
+(Also provide citation for companion paper publication, if available)
+
+To cite this code when you copy or adapt from this code, please include the original URL you copied from and date of retrieval as a comment in your code. Additional information on how to cite can be found in the [ITS CodeHub FAQ](https://its.dot.gov/code/#/faqs).
```
which original URL? Original URL of the source code?

chuehlien replied:
Yes- original URL where one copied the source code from. This is following the guidance at https://integrity.mit.edu/handbook/writing-code and allows for more specific attribution than DOI.

eberlyn replied:
I suggest making slight change to wording - "original URL you copied the source code from..."

eberlyn commented:
Make sure we use the same format as the rest of the template - _italicized test for _instructions, then "Example" and then an example.__

chuehlien committed:
Update Acknowledgements section to use consistent formatting

Update Acknowledgements section to use consistent formatting and update wordings for clarity.

chuehlien committed:
Use blockquote to highlight sample citation

chuehlien commented:
> Make sure we use the same format as the rest of the template - _italicized test for _instructions, then "Example" and then an example.__

Thank you for your review and helpful feedback! The requested changes have been made - please review when you get a chance.

eberlyn commented on README.md:
```diff
@@ -136,12 +136,22 @@ Contact Name: ITS JPO
+
+*Sample citation should be in the below format, with the `formatted fields` replaced with details of your source code*
+
+`author_surname_or_organization`, `first_initial`. (`year`). _`program_or_source_code_title`_ (`code_version`) [Source code]. Provided by ITS CodeHub through GitHub.com. Accessed YYYY-MM-DD from `doi_url`.
```
Sorry, last comment. I think this citation format should be in italics, since it is an instruction to the repo owner, not an example of a filled out README template.

chuehlien replied:
No worries! I wasn't sure if I should italicize it since the source code title portion of the citation is italicized, but agree that it might be more clear if we are consistently italicizing the instructions.

eberlyn replied:
So in that case, keep it all in italics _except_ the 'program or source code title' - make that plain text

chuehlien replied:
Good point - Done!

chuehlien committed:
Italicize citation format.

chuehlien committed:
Make program_or_source_code_title in citation instruction plaintext

chuehlien committed:
Added level 2 headers under Acknowledgements
